engine/dlib/src/dlib/profile/profile_remotery.cpp

```diff
@@ -35,6 +35,7 @@ namespace dmProfile
     static void*                    g_PropertyTreeCallbackCtx = 0;
     static int32_atomic_t           g_ProfilerInitialized = 0;
     static dmSpinlock::Spinlock     g_ProfilerLock;
+    uint8_t g_ProfilerGeneration = 0;
```

Comment:
this variable used to know when static properties and scopes should be recreated